backend/api/Areas/Admin/Controllers/ParcelController.cs

```diff
@@ -53,73 +49,72 @@ public ParcelController(ILogger<ParcelController> logger, IAdminParcelService ad
         /// <param name="quantity"></param>
         /// <param name="sort"></param>
         /// <returns>Paged object with an array of parcels.</returns>
-        [HttpGet("/api/[area]/parcels")]
-        public IActionResult GetParcels(int page = 1, int quantity = 10, string sort = null) // TODO: sort and filter.
+        [HttpGet ("/api/[area]/parcels")]
+        public IActionResult GetParcels (int page = 1, int quantity = 10, string sort = null) // TODO: sort and filter.
         {
             if (page < 1) page = 1;
             if (quantity < 1) quantity = 1;
             if (quantity > 50) quantity = 50;
 
-            var entities = _adminParcelService.GetParcels(page, quantity, sort);
-            var total = entities.Count();
-            var parcels = _mapper.Map<Api.Models.Parts.ParcelModel[]>(entities);
-            var paged = new Pims.Api.Models.Paged<Api.Models.Parts.ParcelModel>(parcels, page, quantity, total);
+            var result = _pimsAdminService.Parcel.GetNoTracking (page, quantity, sort);
+            var entities = _mapper.Map<Api.Models.Parts.ParcelModel[]> (result.Items);
```

Comment:
/nit I was under the impression that the convention for using *entities* was that you were working with a EF Entity, Ie from Pims.Dal.Entities

in this case it is being used to refer to the DTO object.

Reply:
It’s common to have non EF objects in the shared resource.  This specific one is required to handle passing correctly.

Reply:
In some cases the Entity objects are not even shared, only Models.